python/ray/dag/tests/experimental/test_accelerated_dag.py

```diff
@@ -175,6 +181,129 @@ def test_regular_args(ray_start_regular):
     compiled_dag.teardown()
 
 
+def test_multi_args_basic(ray_start_regular):
+    a1 = Actor.remote(0)
+    a2 = Actor.remote(0)
+    a3 = Actor.remote(0)
+    with InputNode() as i:
+        branch1 = a1.inc.bind(i[0])
+        branch2 = a2.inc.bind(i[1])
+        dag = a3.inc_two.bind(branch1, branch2)
```

Comment:
Nice tests! However, could you change the tested method to concatenate the results or something like that? When we use addition, we can't tell from the end result if the args were passed in the correct order.